src/Containers/KitchenStatusModel/KitchenStatusModel.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+import { KitchenStatusModel, KitchenStatusModelProps } from '../../index';
+import { statusEnum} from './StatusButton';
+import { createStoryTitle} from '../../Constants';
+
+export default {
+    title: createStoryTitle('KitchenStatusModel'),
+    component: KitchenStatusModel,
+    argTypes: {
+        onPauseClick: {action: 'Paused For'},
+        onStatusClick: {action: 'Current status'}
+    },
+    args: {
+        minuteAmounts: [10, 20, 30, 40],
+        statusBarColorArray: ['green', 'yellow', 'red'],
+        statusHeaderArray: ['Normal', 'Busy', 'Pause'],
+        statusBodyArray: ['Business as usual', 'Need more prep time', 'Pause new orders'],
```

Comment:
Architecture is incorrect, some of these data types are tied to each other and the structure should reflect that

Reply:
Ex: Normal and Color